src/CustomIndexing/CustomIndexing.jl

```diff
@@ -0,0 +1,130 @@
+module CustomIndexing
```

Comment:
Consider adding a docstring on the module describing its intent. Also missing a license header.